default/stringtables/en.txt

```diff
@@ -1915,6 +1915,9 @@ Automatically quits once any turns specified by --auto-advance-n-turns are compl
 OPTIONS_DB_BINARY_SERIALIZATION
 Use Binary serialization for saving games. Binary serialization is faster to save and load, but may not be possible to load on a different operating system.
 
+OPTIONS_DB_SERVER_BINARY_SERIALIZATION
+Use Binary serialization for client-server interaction. Binary serialization is faster to transfer, but may not be possible to read message on a different operating system and architectures.
```

Comment:
It looks to me like this is purely a server-side option, and this help message should probably indicate that.

Reply:
Fixed.